src/python/pants/backend/python/targets/python_distribution.py

```diff
@@ -57,10 +57,6 @@ def __init__(self,
   def has_native_sources(self):
     return self.has_sources(extension=tuple(self.native_source_extensions))
 
-  @property
-  def platforms(self):
-    return ['current']
```

Comment:
Note that this is part of removing `PythonDistribution` from `may_have_explicit_python_platform()` in `pex_build_util.py`.